### PR TITLE
E110 cookie

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

File: src/service/auth.js [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      signature,
+      redirectUrl,
+    })
+    .then(res => {
+      if (!res.success) throw new Error(res.message);
+
+      return res;
+    });
+
+const getSeedTemplateUrl = () => `${identityBaseUrl}/auth/recover/template`;
```
Comment (lamartire) [inline src/service/auth.js]: It can be just constant and should be defined at the top of file

File: src/store/modules/AuthModule.js [hunk 2]
```diff
@@ -185,6 +197,8 @@ class AuthModule extends VuexModule {
 
   @Action
   logout() {
+    this.cookieExpire.value.updateExpireAt(0);
```
Comment (lamartire) [inline src/store/modules/AuthModule.js]: I think better to rename it to `setExpirationTime`

File: src/store/modules/CoreModule.js [hunk 3]
```diff
@@ -88,6 +114,18 @@ class CoreModule extends VuexModule {
...
+  @Action
+  cancelAllChannels() {
+    const fail = () => Answer.createFail(ERRORS.AUTH_CANCELED_BY_USER);
+
+    permissionChannel.put(fail());
+    authChannel.put(fail());
+    accountChannel.put(fail());
+    signChannel.put(fail());
+    documentChannel.put(fail());
+    walletChannel.put(fail());
```
Comment (lamartire) [inline src/store/modules/CoreModule.js]: `[permissionChannel, authChannel, accountChannel, signChannel, documentChannel, walletChannel].forEach(channel => channel.put(fail))` 🙃

File: tests/unit/specs/components/forms/Code/OtpCode/RecoverSmsForm.spec.js [hunk 4]
```diff
@@ -68,9 +68,7 @@ describe('RecoverSmsForm', () => {
         const errorPhoneNotExist = new Error(error.message);
         errorPhoneNotExist.code = 400;
 
-        identityService.sendOtpRecoverSms.mockRejectedValueOnce(
-          errorPhoneNotExist,
-        );
+        authService.sendOtpRecoverSms.mockRejectedValueOnce(errorPhoneNotExist);
```
Comment (lamartire) [inline tests/unit/specs/components/forms/Code/OtpCode/RecoverSmsForm.spec.js]: Something wrong with formatting

Comment (lamartire) [inline tests/unit/specs/components/forms/Code/OtpCode/RecoverSmsForm.spec.js]: Something wrong with formatting

File: tests/unit/specs/service/auth.spec.js [hunk 5]
```diff
@@ -53,19 +53,19 @@ describe('identity service', () => {
...
 
     it('should handle rejected GET /auth/recover request', async () => {
       expect.assertions(1);
 
       axiosMock.onGet(url).reply(500);
 
-      await expect(
-        identityService.getRecoveryIdentifier(email),
-      ).rejects.toThrow(expect.any(Error));
+      await expect(authService.getRecoveryIdentifier(email)).rejects.toThrow(
```
Comment (lamartire) [inline tests/unit/specs/service/auth.spec.js]: Does it really work correctly?

File: src/class/CookieExpire.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+import EventEmitter from '@endpass/class/EventEmitter';
+
+const EXPIRE_COOKIE_TIME_KEY = 'endpass--expired-time';
+const EXPIRE_CHECK_TIMEOUT = 2000;
+
+const EVENT_COOKIE_EXPIRED = 'cookie-expire';
+
+export default class CookieExpire {
+  constructor() {
+    this.intervalId = undefined;
```
[the remark this answers is not in the document]
Comment (dudiq) [reply src/class/CookieExpire.js]: TS check are not passed with null

File: tests/unit/specs/service/auth.spec.js (hunk 5 above)
Comment (dudiq) [reply tests/unit/specs/service/auth.spec.js]: what do you mean?

File: src/service/auth.js (hunk 1 above)
Comment (dudiq) [reply src/service/auth.js]: main target was split `auth` parts from `identity` for more readable.
that's why some methods not refactored and stay as is.

**Pull request merged**